### PR TITLE
Provision for tagging

### PR DESCRIPTION
**Pull request by raiyankamal**
Solution to #14

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -104,6 +119,10 @@ Can also be provided as the environment variable `BUCKET_NAME`.
 An optional folder to stream log files to. Takes a path string,
 eg: "my/subfolder" or "nested".
 
+#### folder
```
Comment (autopulated) [inline readme.md]: this should be `#### tags` presumably :)

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "s3-streamlogger",
-  "version": "1.2.0",
+  "version": "1.2.1",
```
Comment (autopulated) [inline package.json]: please don't bump the version in PRs – I'll bump the version just before tagging the release

**Comment by autopulated** [comment]
Thanks! This mostly looks good, just see the comments above. Also, do you know which version of the AWS SDK introduced support for tagging? (our current AWS SDK dependency allows versions which I think would be too old, so that probably needs updating to the version where tagging was introduced)

File: readme.md (hunk 1 above)
Comment (raiyankamal) [reply readme.md]: You're right. I copied the folders section but forgot to change the section title.

File: package.json (hunk 2 above)
Comment (raiyankamal) [reply package.json]: No problem. I'll send a new commit with these changes soon.

**Comment by raiyankamal** [comment]
Thank for the review. A new commit with the requested changes is already pushed.

Regarding the AWS SDK version, I believe its [2.80.0](https://github.com/aws/aws-sdk-js/blob/master/CHANGELOG.md#2800).

**Comment by autopulated** [comment]
👍 Thanks!

**Comment by autopulated** [comment]
published in v1.3.0

**Pull request merged**